bot/exts/info/codeblock/_instructions.py

```diff
@@ -135,12 +126,8 @@ def _get_no_lang_message(content: str) -> str | None:
         example_blocks = _get_example("py")
 
         # Note that _get_bad_ticks_message expects the first line to have two newlines.
-        return (
-            "It looks like you pasted Python code without syntax highlighting.\n\n"
-            "Please use syntax highlighting to improve the legibility of your code and make "
-            "it easier for us to help you.\n\n"
-            f"**To do this, use the following method:**\n{example_blocks}"
-        )
+        return f"Please add a `py` after the three backticks.\n\n{example_blocks}"
```

Comment:
The embed title already says "please", so saying it again feels redundant to me.